randovania/gui/main_window.py

```diff
@@ -539,9 +540,14 @@ def _open_auto_tracker(self):
 
     def _on_menu_action_previously_generated_games(self):
         path = self._options.data_dir.joinpath("game_history")
-        if platform.system() == "Windows":
-            os.startfile(path)
-        else:
+        try:
+            if platform.system() == "Windows":
+                os.startfile(path)
+            elif platform.system() == "Darwin":
+                subprocess.Popen(["open"], path)
```

Comment:
This is likely supposed to be `["open", path]`, no?

Reply:
Yep. I'm going to try to test this on an old macbook to make sure the macOS stuff works, as well as make sure the test covers macOS